torchci/lib/bot/autoLabelBot.ts

```diff
@@ -12,14 +12,25 @@ import {
   LabelToLabelConfigTracker,
 } from "./utils";
 
-const titleRegexToLabel: [RegExp, string][] = [
+// List of regex patterns for assigning labels to both Pull Requests and Issues
+const IssueAndPRRegexToLabel: [RegExp, string][] = [
   [/rocm/gi, "module: rocm"],
   [/rocm/gi, "ciflow/rocm"],
   [/vulkan/gi, "module: vulkan"],
-  [/vulkan/gi, "ciflow/periodic"], // Vulkan tests are run periodically
   [/DISABLED\s+test.*\(.*\)/g, "skipped"],
```

Comment:
disabled should be in the Issue title list, ciflow/rocm should be in the pr title list, and I think Huy mentioned that he got rid of vulkan

Reply:
there are two DISABLEDs, assume you say that all of them should move to Issue title list?